app/helpers/home_helper.rb

```diff
@@ -1,23 +1,36 @@
 module HomeHelper
   def map_meta
     if @user
-      title "#{@user.short_name}'s Suggestions"
-      meta_description "Check out where #{@user.first_name} thinks we should go. Vote for the best ideas."
+      user_meta
     elsif @experience
-      title "#{@experience.name}"
-      meta_description "Should we do this? Cast your vote."
+      experience_meta
     else
-      title 'The Map'
-      meta_description 'Suggest our next stop, and vote for other destinations.'
+      default_meta
     end
   end
 
+  def user_meta
+    title "#{@user.short_name}'s Suggestions"
+    meta_description "Check out where #{@user.first_name} thinks we should \
+      go. Vote for the best ideas."
+  end
+
+  def experience_meta
+    title "#{@experience.name}"
+    meta_description 'Should we do this? Cast your vote.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.